### ISSUE TITLE
generate_rbs_for_models raises error Could not find table 'action_text_rich_texts'

### ISSUE THREAD

**Issue by k-maekawa** [pocke/rbs_rails#81]
### Summary

`Could not find table 'action_text_rich_texts' ActiveRecord::StatementInvalid Error` happens in  new simple rails project without model.

### Gemfile

```ruby
source 'https://rubygems.org'
git_source(:github) { |repo| "https://github.com/#{repo}.git" }

ruby '3.0.0'

# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '~> 6.1.0'
# Use sqlite3 as the database for Active Record
gem 'sqlite3', '~> 1.4'
# Use Puma as the app server
gem 'puma', '~> 5.0'
# Use SCSS for stylesheets
gem 'sass-rails', '>= 6'
# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
gem 'webpacker', '~> 5.0'
# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
gem 'turbolinks', '~> 5'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.7'
# Use Redis adapter to run Action Cable in production
# gem 'redis', '~> 4.0'
# Use Active Model has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Active Storage variant
# gem 'image_processing', '~> 1.2'

# Reduces boot times through caching; required in config/boot.rb
gem 'bootsnap', '>= 1.4.4', require: false

gem 'rbs_rails', require: false

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
end

group :development do
  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
  gem 'web-console', '>= 4.1.0'
  # Display performance information such as SQL time and flame graphs for each request in your browser.
  # Can be configured to work on production as well see: https://github.com/MiniProfiler/rack-mini-profiler/blob/master/README.md
  gem 'rack-mini-profiler', '~> 2.0'
  gem 'listen', '~> 3.3'
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
end

group :test do
  # Adds support for Capybara system testing and selenium driver
  gem 'capybara', '>= 3.26'
  gem 'selenium-webdriver'
  # Easy installation and use of web drivers to run system tests with browsers
  gem 'webdrivers'
end

# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]

```

### Logs

```ruby
$ bin/rake rbs_rails:generate_rbs_for_models
Running via Spring preloader in process 29758
rake aborted!
ActiveRecord::StatementInvalid: Could not find table 'action_text_rich_texts'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/sqlite3_adapter.rb:346:in `table_structure'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract/schema_statements.rb:116:in `columns'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/schema_cache.rb:104:in `block in columns'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/schema_cache.rb:103:in `fetch'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/schema_cache.rb:103:in `columns'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/schema_cache.rb:112:in `block in columns_hash'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/schema_cache.rb:111:in `fetch'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/schema_cache.rb:111:in `columns_hash'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/model_schema.rb:553:in `load_schema!'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/attributes.rb:250:in `load_schema!'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/model_schema.rb:539:in `block in load_schema'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/model_schema.rb:536:in `synchronize'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/model_schema.rb:536:in `load_schema'
sandbox/.bundle/ruby/3.0.0/gems/activerecord-6.1.0/lib/active_record/model_schema.rb:397:in `columns'
sandbox/.bundle/ruby/3.0.0/gems/rbs_rails-0.6.0/lib/rbs_rails/active_record.rb:266:in `columns'
sandbox/.bundle/ruby/3.0.0/gems/rbs_rails-0.6.0/lib/rbs_rails/active_record.rb:25:in `klass_decl'
sandbox/.bundle/ruby/3.0.0/gems/rbs_rails-0.6.0/lib/rbs_rails/active_record.rb:14:in `generate'
sandbox/.bundle/ruby/3.0.0/gems/rbs_rails-0.6.0/lib/rbs_rails/active_record.rb:4:in `class_to_rbs'
sandbox/.bundle/ruby/3.0.0/gems/rbs_rails-0.6.0/lib/rbs_rails/rake_task.rb:58:in `block (2 levels) in def_generate_rbs_for_models'
sandbox/.bundle/ruby/3.0.0/gems/rbs_rails-0.6.0/lib/rbs_rails/rake_task.rb:51:in `each'
sandbox/.bundle/ruby/3.0.0/gems/rbs_rails-0.6.0/lib/rbs_rails/rake_task.rb:51:in `block in def_generate_rbs_for_models'
sandbox/.bundle/ruby/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:59:in `load'
sandbox/.bundle/ruby/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:59:in `load'
sandbox/.bundle/ruby/3.0.0/gems/activesupport-6.1.0/lib/active_support/fork_tracker.rb:10:in `block in fork'
sandbox/.bundle/ruby/3.0.0/gems/activesupport-6.1.0/lib/active_support/fork_tracker.rb:10:in `block in fork'
sandbox/.bundle/ruby/3.0.0/gems/activesupport-6.1.0/lib/active_support/fork_tracker.rb:8:in `fork'
sandbox/.bundle/ruby/3.0.0/gems/activesupport-6.1.0/lib/active_support/fork_tracker.rb:8:in `fork'
sandbox/.bundle/ruby/3.0.0/gems/activesupport-6.1.0/lib/active_support/fork_tracker.rb:26:in `fork'
sandbox/.bundle/ruby/3.0.0/gems/activesupport-6.1.0/lib/active_support/fork_tracker.rb:8:in `fork'
sandbox/.bundle/ruby/3.0.0/gems/activesupport-6.1.0/lib/active_support/fork_tracker.rb:26:in `fork'
<internal:.rbenv/versions/3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
<internal:.rbenv/versions/3.0.0/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
-e:1:in `<main>'
Tasks: TOP => rbs_rails:generate_rbs_for_models
(See full trace by running task with --trace)
```

https://github.com/k-maekawa/rbs_sandbox

**Comment by k-maekawa** [comment]
@pocke 

I found ignore_model_if attribute. So, I defined `rbs.rake` 

```ruby
require 'rbs_rails/rake_task'

RbsRails::RakeTask.new do |rbs|
  rbs.ignore_model_if = ->(model) { [ActionText::RichText, ActionMailbox::InboundEmail, ActiveStorage::Blob, ActiveStorage::VariantRecord, ActiveStorage::Attachment].include?(model) }
end
```

Do we have to initialize `RbsRails::RakeTask` as above, when we want to ignore these models which rails gives by default but we do not  necessarily  use.

```ruby
irb(main):002:0> ActiveRecord::Base.descendants
=> [
ApplicationRecord(abstract), 
ActionText::Record(abstract),
ActionText::RichText (call 'ActionText::RichText.connection' to establish a connection),
ActionMailbox::Record(abstract),
ActionMailbox::InboundEmail (call 'ActionMailbox::InboundEmail.connection' to establish a connection), ActiveStorage::Record(abstract),
ActiveStorage::Blob (call 'ActiveStorage::Blob.connection' to establish a connection),
ActiveStorage::VariantRecord (call 'ActiveStorage::VariantRecord.connection' to establish a connection), ActiveStorage::Attachment (call 'ActiveStorage::Attachment.connection' to establish a connection)]
```

**Comment by pocke** [comment]
> Could not find table 'action_text_rich_texts' ActiveRecord::StatementInvalid Error happens in new simple rails project without model.

**Comment by pocke** [comment]
I've released RBS v0.8.1. It ignores AR classes if the class doesn't have a table in DB.
I think it solves this issue without the configuration. So I closed this issue.


If you have any opinion about the interface of the rake task, feel free to commit to this issue or open a new issue.

Thanks.
